src/allotropy/parsers/agilent_gen5/agilent_gen5_structure.py

```diff
@@ -3,23 +3,24 @@
 from dataclasses import dataclass
 
 from allotropy.exceptions import AllotropeConversionError
+from allotropy.parsers.agilent_gen5.constants import MULTIPLATE_FILE_ERROR
 from allotropy.parsers.agilent_gen5.plate_data import PlateData
 from allotropy.parsers.agilent_gen5.section_reader import SectionLinesReader
 
 
 @dataclass(frozen=True)
 class Data:
-    plates: list[PlateData]
+    plate_data: PlateData
 
     @staticmethod
     def create(section_lines_reader: SectionLinesReader) -> Data:
-        plates: list[PlateData] = [
-            PlateData.create(lines_reader)
-            for lines_reader in section_lines_reader.iter_sections("^Software Version")
-        ]
+        plate_readers = list(section_lines_reader.iter_sections("^Software Version"))
```

Comment:
Can you change this back to `plates` or perhaps `plate_sections`? 

`plate_readers` makes it sound like these are reader classes, which could be confusing.

Reply:
I changed it to plate_readers because `SectionLinesReader.iter_sections` yields `LinesReader` instances. But I agree it may be not the best wording for it.